### ISSUE TITLE
GraphQL SortOrder enum maybe not working properly

### ISSUE THREAD

**Issue by jimutt** [gridsome/gridsome#59]
Hi!
I'm having some issues with using the "order" parameter for querying a local Markdown source. Whenever I try to use it with the DESC or ASC enum I get the following error:  `Variable \"$_v1_order\" got invalid value -1; Expected type SortOrder.`

Have I done something wrong? Is this a known issue or should I investigate it further on my side and try to reproduce it in a minimal project? 

![screenshot from 2018-11-28 22 56 34](https://user-images.githubusercontent.com/1392559/49185036-1edd5880-f361-11e8-8ecf-6d69534eef8c.png)

**Comment by hjvedvik** [comment]
You are not doing it wrong. Looks like the `mergeSchemas` function which Gridsome use from `graphql-tools` isn't handling all enum values correctly. But I managed to find a workaround so I'll publish a fix as soon as possible :)

**Comment by jimutt** [comment]
@hjvedvik  What's the status on this one? I just tried with 0.35 and it seems to be working (not getting any errors when using `order`). But when I try to order on my custom `publishedDate` property as above (it's correctly specified as a Date type in the schema) the ordering does not work properly. I suspect that it maybe sorts the string representation of the dates instead of the actual date values?

**Comment by hjvedvik** [comment]
Yes, it was fixed in `0.3.5` :) Unfortunately, it's not possible to sort on custom fields yet because of a limitation in the in-memory database Gridsome use. But I saw a PR a few days ago which makes it possible, so we are waiting for them to publish an update.

**Comment by jimutt** [comment]
@hjvedvik Ah, alright! Thanks for the information

**Comment by jimutt** [comment]
@hjvedvik  What is the status for sorting on custom fields? Is it something you plan to implement in the near future? :-) It feels quite limiting only being able to sort content on the default fields.
